src/LivetimeCounter.cpp

```diff
@@ -0,0 +1,93 @@
+#include "trigger/LivetimeCounter.hpp"
+
+#include "logging/Logging.hpp"
+
+#include <sstream>
+
+namespace dunedaq::trigger {
+
+LivetimeCounter::LivetimeCounter(LivetimeCounter::State state)
+  : m_state(state)
+  , m_last_state_change_time(now())
```

Comment:
Another issue, the system time is not the right measure for live time.  The data clock is.